### PR TITLE
Add metadata

### PR DESCRIPTION
**Pull request by mjseaman**
Add metadata, so that license activation appears in the side menu

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/enterprise/activate-license.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-
++++
+title = "License Activation"
```
Comment (osg-grafana) [inline docs/sources/enterprise/activate-license.md:2]: License activation

(Use sentence style for headings)

**Review by osg-grafana — CHANGES_REQUESTED** [review]
Use sentence style headings.

File: docs/sources/enterprise/activate-license.md (hunk 1 above)
Comment (mjseaman) [reply docs/sources/enterprise/activate-license.md]: Updated

**Review by osg-grafana — APPROVED** [review]

File: docs/sources/enterprise/activate-license.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Follow these steps to activate your Grafana Enterprise license:
 
 ## Step 1. Download your license file
 
-To download your Grafana Enterprise license, log in to your [Grafana Cloud Account](https://grafana.com) and go to your **Org Profile**. In the side menu there is a section for Grafana Enterprise licenses. At the bottom of the license details page there is **Download Token** link that will download the license.jwt file containing your license.
+To download your Grafana Enterprise license, log in to your [Grafana Cloud Account](https://grafana.com) and go to your **Org Profile**. In the side menu there is a section for Grafana Enterprise licenses. At the bottom of the license details page there is a **Download Token** link that will download the license.jwt file containing your license.
```
Comment (osg-grafana) [inline docs/sources/enterprise/activate-license.md:14]: 1. Sign in to your [Grafana Cloud Account](https://grafana.com).
2. Go to your **Org Profile**.
    In the side menu, you can see a section for Grafana Enterprise licenses.
3. From the bottom of the license details page, select **Download token** to download the `license.jwt` file that contains your license.

**Review by osg-grafana — DISMISSED** [review]

**Review by oddlittlebird — APPROVED** [review]

**Comment by mjseaman** [comment]
Hi @oddlittlebird I'm just seeing this after a while - have these changes been merged; should I close this?

**Comment by oddlittlebird** [comment]
I don't know if they've been merged, but I doubt they have. Please try to merge, it's a good change.

**Comment by daniellee** [comment]
@mjseaman sorry for this being so slow. There was a tricky merge conflict to resolve due to a file being renamed.

File: docs/sources/enterprise/license/activate-license.md [hunk 3]
```diff
@@ -17,35 +18,50 @@ To download your Grafana Enterprise license:
 1. Go to **My Account** and select an organization from the drop-down menu at the top left of the page. On the Overview page for each organization, you can see a section for Grafana Enterprise licenses. Click **Details** next to a license.
 1. At the bottom of the license details page, select **Download token** to download the license.jwt file that contains your license.
 
-## Step 2. Add your license file to a Grafana instance
+## Step 2. Add your license to a Grafana instance
 
-There are two different ways to add the license file to a Grafana instance:
+There are three different ways to add the license to a Grafana instance:
```
Comment (osg-grafana) [inline docs/sources/enterprise/license/activate-license.md]: Let's generalize this and remove the baked in number:
```suggestion
There is more than one way to add the license to a Grafana instance:
```

File: docs/sources/enterprise/license/activate-license.md [hunk 4]
```diff
@@ -17,35 +18,50 @@ To download your Grafana Enterprise license:
...
 
-- Option 1: Upload the license file through the Grafana Server Administrator page
+### Upload the license file through the Grafana Server Administrator page
```
Comment (osg-grafana) [inline docs/sources/enterprise/license/activate-license.md]:
```suggestion
### Upload the license file via the Grafana server administrator page
```

File: docs/sources/enterprise/license/activate-license.md [hunk 5]
```diff
@@ -17,35 +18,50 @@ To download your Grafana Enterprise license:
...
-  1. Navigate to **Server Admin > Upgrade** within Grafana.
-  1. Click **Upload license token file**.
-  1. Select your license file, and upload it.
+This is the preferred option for single instance installations of Grafana Enterprise.
 
-- Option 2: Place the _license.jwt_ file in Grafana's data folder.
+1. Sign in as a Grafana server admin.
+1. Navigate to **Server Admin > Upgrade** within Grafana.
+1. Click **Upload license token file**.
+1. Select your license file, and upload it.
```
Comment (osg-grafana) [inline docs/sources/enterprise/license/activate-license.md]:
```suggestion
1. Sign in as a Grafana server administrator.
1. Navigate to **Server Admin > Upgrade** within Grafana.
1. Click **Upload license token file**.
1. Select your license file, and upload it.
```

File: docs/sources/enterprise/license/activate-license.md [hunk 6]
```diff
@@ -17,35 +18,50 @@ To download your Grafana Enterprise license:
...
 
-  This is usually located at `/var/lib/grafana` on Linux systems.
+### Place the license.jwt file in Grafana's data folder
```
Comment (osg-grafana) [inline docs/sources/enterprise/license/activate-license.md]:
```suggestion
### Put the `license.jwt` file into the data directory of Grafana
```

File: docs/sources/enterprise/license/activate-license.md [hunk 7]
```diff
@@ -17,35 +18,50 @@ To download your Grafana Enterprise license:
...
 
-  You can also configure a custom location for the license file using the grafana.ini setting:
+The data folder is usually `/var/lib/grafana` on Linux systems.
```
Comment (osg-grafana) [inline docs/sources/enterprise/license/activate-license.md]:
```suggestion
On Linux systems, the data directory is usually at `/var/lib/grafana`.
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]

File: docs/sources/enterprise/license/activate-license.md [hunk 8]
```diff
@@ -14,10 +14,9 @@ Follow these steps to activate your Grafana Enterprise license:
 
 To download your Grafana Enterprise license:
 
-1. Log in to your [Grafana Cloud Account](https://grafana.com).
-1. Go to your **Org Profile**.
-1. Go to the section for Grafana Enterprise licenses in the side menu.
-1. At the bottom of the license details page there is **Download Token** link that will download the _license.jwt_ file containing your license to your computer.
+1. Sign in to your [Grafana Cloud](https://grafana.com) account.
+1. Go to **My Account** and select an organization from the drop-down menu at the top left of the page. On the Overview page for each organization, you can see a section for Grafana Enterprise licenses. Click **Details** next to a license.
+1. At the bottom of the license details page, select **Download token** to download the license.jwt file that contains your license.
```
Comment (osg-grafana) [inline docs/sources/enterprise/license/activate-license.md:19]:
```suggestion
1. At the bottom of the license details page, select **Download token** to download the `license.jwt` file that contains your license.
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]

**Review by osg-grafana — APPROVED** [review]

File: docs/sources/enterprise/license/activate-license.md (hunk 3 above)
Comment (daniellee) [reply docs/sources/enterprise/license/activate-license.md]: Added manually. (Had messed up the first merge conflict so couldn't apply the suggestion automatically)

File: docs/sources/enterprise/license/activate-license.md (hunk 4 above)
Comment (daniellee) [reply docs/sources/enterprise/license/activate-license.md]: Added manually.

File: docs/sources/enterprise/license/activate-license.md (hunk 5 above)
Comment (daniellee) [reply docs/sources/enterprise/license/activate-license.md]: Added manually.

File: docs/sources/enterprise/license/activate-license.md (hunk 7 above)
Comment (daniellee) [reply docs/sources/enterprise/license/activate-license.md]: Added manually

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**